frontend/talentsearch/src/js/components/signAndSubmit/SignAndSubmitPage.tsx

```diff
@@ -0,0 +1,284 @@
+import * as React from "react";
+import { Link } from "@common/components";
+import { Input, Submit } from "@common/components/form";
+import NotFound from "@common/components/NotFound";
+import Pending from "@common/components/Pending";
+import TableOfContents from "@common/components/TableOfContents";
+import { commonMessages, errorMessages } from "@common/messages";
+import { ArrowSmRightIcon } from "@heroicons/react/outline";
+import {
+  BriefcaseIcon,
+  ClipboardCheckIcon,
+  UserIcon,
+} from "@heroicons/react/solid";
+import { uniqueId } from "lodash";
```

Comment:
I think the preferred way of importing lodash is:

```tsx
import uniqueId from "lodash/uniqueId";
```

Reply:
Fixed by [Fix uniqueId lodash import](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3678/commits/54d52434c33723b6a09f1d74b30227a262f0977f)